aims/aim3.md

```diff
@@ -0,0 +1,60 @@
+These models have engineered features that encode information about gene ontology terms and KEGG pathways.
+We have shown that these models capture more information than linear models like PCA, ICA, and NMF, and that the learned features are often overlapping and redundant [7].
+We have also recently trained a VAE on cancer gene expression datasets and observed that the learned latent features capture interesting biological distributions of features across different tumors [8].
+Therefore, we are confident that modeling gene expression with reconstruction cost captures meaningful biology.
+Furthermore, we trained VAEs and GANs on electronic health record (EHR) data including a model to generate privacy aware fake datasets to supplement learning or data sharing [9,10].
+We will develop similar approaches to extract knowledge from Human Cell Atlas (HCA) single cell data.
+
+## Enabling latent space arithmetic in single cell gene expression with variational autoencoders:
+
+Once a VAE is trained on HCA single cell gene expression data, we will obtain the latent space representation of a heterogeneous population of blood cells.
+We will use well characterized gene expression markers and unsupervised clustering algorithms to define core subsets of common blood cell types.
+We will define core subsets by positive silhouette widths of well characterized cell types representing a variety of the differentiation trajectories of hematopoiesis.
+We will test our hypothesis that the learned latent space vectors can be manipulated mathematically to obtain biological insights and useful properties.
+We will calculate the mean latent space vector representing erythroid cells and mean vector representing the population of stem cell progenitors.
+We will take the mean of these two vectors, which, if the hypothesis were true, will represent cells in intermediate trajectories.
+We will repeat this procedure with mature leukocyte populations.
+We will determine how similar the observed mean vectors for observed intermediate cell-types are to the predicted cell types.
+As a negative control, we will perform the same arithmetic using raw gene expression features, PCA features, and vanilla autoencoder features
+A manifold preserving algorithm, a VAE can capture underlying trajectories between cell states, and can help to define core cell types.
+
+## Potential Pitfalls and Alternative Approaches:
+
+-	Latent space does not capture manifold
+    - Compare differentiation trajectory to current state of the art pseudotime algorithms
+-	Cell types not defined
+    -	Use single cell markers as identified in Villani et al. 2017 and conservative unsupervised clustering algorithms to define core subset [11]
+-	Adjusting for Large batch effects
+    -	Variance captured in data should be explained by VAE feature(s)
+    - Identify highest activated feature separating batch, set activation nodes constant for each cell, and test reconstruction pre and post correction
```

Comment:
This will probably require additional testing using data with well characterized batch issues and benchmarked against current methods. Could be particularly fruitful in single cell data. But this may fit  closer with @ejfertig thoughts on batch correction